src/common/transformations/include/transformations/common_optimizations/nop_elimination.hpp

```diff
@@ -157,7 +159,29 @@ class ov::pass::NopSliceBeforeGatherElements : public ov::pass::MatcherPass {
 };
 
 /**
- * @ingroup ov_transformation_common_api
+ * @ingroup ie_transformation_common_api
```

Comment:
```suggestion
 * @ingroup ov_transformation_common_api
```

Reply:
Thanks for comment! Will fix in the next PR :)